groups/groups.yaml

```diff
@@ -1038,6 +1038,18 @@ groups:
       - bowei@google.com
       - antonio.ojea.garcia@gmail.com
 
+  - email-id: k8s-infra-staging-slack-tools@kubernetes.io
+    name: k8s-infra-staging-slack-tools
+    description: |-
+      ACL for staging Slack Tools
+    settings:
+      ReconcileMembers: "true"
+    owners:
```

Comment:
Also @jeefy ?

Reply:
@ameukam Could you give me his email please?

Reply:
Can be added later.